frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveExport/ExportBanner/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+// Default dimensions
+export const DATA_MESSAGE_BANNER_WIDTH_PX = 433;
+export const DATA_MESSAGE_BANNER_HEIGHT_PX = 64;
+
+interface Props {
+  height?: number;
+  width?: number;
+  centered?: boolean;
+}
+
+export const UnderlyingDataChangeBanner = React.memo(
+  function UnderlyingDataChangeMessage({
+    height = DATA_MESSAGE_BANNER_HEIGHT_PX,
+    width = DATA_MESSAGE_BANNER_WIDTH_PX,
+    centered = true,
+  }: Props): JSX.Element {
+    return (
+      <svg
+        id="data-message-banner"
+        fill="none"
+        width={width}
+        height={height}
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox={`0 0 ${width} ${height}`}
+      >
+        <rect width="100%" height="100%" rx="4" fill="#F8F8F8" />
+        <path fill="#767676" d={centered ? CENTERED_TEXT : LEFT_ALIGNED_TEXT} />
+      </svg>
+    );
+  }
+);
+
+const CENTERED_TEXT =
```

Comment:
Text is rendered as an svg path element, so we have separate path data depending on the context